### PR TITLE
default_app_config deprecated in Django 3.2

### PR DESCRIPTION
**Pull request by smithdc1**
Not sure if you want a test for this?

### CODE REVIEW

**Review by carltongibson — APPROVED** [review]
No, that's fine. Thank you.

**Comment by smithdc1** [comment]
Hi @carltongibson .

Looks like this broke the rtd build.

Let me have a look. 🔍

**Comment by carltongibson** [comment]
The build probably hasn't needed the dependencies until this change (which probably means I need to adjust it on RTD...)

**Comment by carltongibson** [comment]
I can't resolve this today. Installing Django fails because of a dependency clash with asgiref/django, which is [trac-32128](https://code.djangoproject.com/ticket/32128)

File: channels/__init__.py [hunk 1]
```diff
@@ -1,4 +1,7 @@
+import django
```
Comment (carltongibson) [inline channels/__init__.py:1]: What if we just `try` this?

**Pull request merged**